### DISCUSSION TITLE
How can I include reveal.js presentation in a quarto website ?

### DISCUSSION THREAD

**Discussion by wilcar** [quarto-dev/quarto-cli#7147] [Q&A]
### Description

In this page, you include a reveal.js presentation in your website.
https://quarto.org/docs/presentations/revealjs/

How can I do this ?

**Comment by cscheid** [comment]
In general, you can click on "edit this page" to view the source and see how we do things. In this specific case, we just use an iframe.

**Comment by mcanouil** [comment]
This is a topic discussed already a number of times.
I cannot link them all now, but a quick search on Discussions should highlight them all to you, especially if you filter your search using "revealjs" and "website" labels.

---
Edit: <https://github.com/quarto-dev/quarto-cli/discussions?discussions_q=is%3Aopen+label%3Arevealjs+label%3Awebsites>

<img width="1234" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/bf879a8f-decb-4f93-ba2b-2125867b94a2">
